src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/CourtEventFilter.java

```diff
@@ -33,7 +33,7 @@ public Predicate toPredicate(final Root<CourtEvent> root, final CriteriaQuery<?>
         final ImmutableList.Builder<Predicate> predicateBuilder = ImmutableList.builder();
 
         if (bookingId != null) {
-            predicateBuilder.add(cb.equal(root.get("offenderBooking"), bookingId));
+            predicateBuilder.add(cb.equal(root.get("offenderBooking").get("bookingId"), bookingId));
```

Comment:
this was an odd one that caused issues due to hibernate upgrade. Not sure how it worked previously but need to get the id in order to do the comparison.